x-pack/auditbeat/module/system/user/user.go

```diff
@@ -46,10 +51,35 @@ const (
 	eventActionPasswordChanged = "password_changed"
 )
 
+type passwordType uint8
+
+const (
+	detectionDisabled passwordType = iota + 1
+	shadowPassword
+	passwordDisabled
+	noPassword
+	cryptPassword
+)
+
+func (t passwordType) string() string {
```

Comment:
This should be `String() string`. You are implementing 
https://golang.org/pkg/fmt/#Stringer.